core/src/main/scala/com/dimafeng/testcontainers/implicits/DockerImageNameConverters.scala

```diff
@@ -0,0 +1,10 @@
+package com.dimafeng.testcontainers.implicits
+
+import org.testcontainers.utility.DockerImageName
+
+trait DockerImageNameConverters {
+  implicit def stringToDockerImageName(s: String): DockerImageName =
```

Comment:
Should we annotate this conversion as deprecated?

Reply:
Sure, will add it